<launch>
   <!-- <include file="$(find gps)/launch/run.launch"/> -->
   不同设备需要有不同ID
   <param name="Car_ID" value="Car003"/>   
   <param name="broker" value="192.168.10.101"/>

   <node pkg="mqtt_commx" type="simple_pub.py" name="simple_pub" output="screen" respawn="true">
      <param name="actor" value="follower"/>
   </node>

   <node pkg="mqtt_commx" type="simple_sub.py" name="simple_sub" output="screen" respawn="true"/>
   
   <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="base_map" args="2 1.37 3 0 0 0 base_link map"/> -->
  
</launch>